tests/utils/throttleTick.test.js

```diff
@@ -0,0 +1,77 @@
+/* global assert, suite, test */
+var throttleTick = require('utils').throttleTick;
+
+suite('utils.throttleTick', function () {
+  var ts;
```

Comment:
no, throttleTick should only be called once, the actual callback is the one to look for, and this was easier...